docs/source/usage/iss/iss_cli.sh

```diff
@@ -54,7 +54,7 @@ starfish target_assignment \
     --coordinates-geojson /tmp/starfish/results/regions.geojson \
     --intensities /tmp/starfish/results/spots.nc \
     --output /tmp/starfish/results/targeted-spots.nc \
-    PointInPoly2D
+    Label
```

Comment:
@shanaxel42 isn't this tested as part of unit tests?

Reply:
There's a python version that was updated in https://github.com/spacetx/starfish/commit/b1c20bcef23c23df46ae809c7d31950d8b1f4f1b#diff-82b3014b798b61630569e45cc9b33581R117